zsh-wakatime.plugin.zsh

```diff
@@ -19,4 +23,8 @@ waka_filename() {
     fi
 }
 
-precmd_functions+=(send_wakatime_heartbeat)
+heartbeat() {
+  (send_wakatime_heartbeat > /dev/null 2>&1 &)
```

Comment:
Since `git rev-parse` is slow, and it was pausing my shell for noticeable time. I think we could  remove the one at the end of `wakatime` command